### PR TITLE
@kanaabe => Add cover image support to Canvas video

### PR DESCRIPTION
**Pull request by damassi**
Adds cover image support to canvas video. 

**NOTE!** There is no field in positron for uploading cover images in this way, so pulls from display ad. 

screen shot 2017-11-06 at 3 38 50 pm

### CODE REVIEW

File: src/Components/Publishing/Display/Canvas/CanvasContainer.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export class CanvasContainerComponent extends React.Component<CanvasContainerPro
 
   render() {
     const { campaign, disclaimer, size, unit } = this.props
-    const { assets, layout, link: { url } } = unit
+    const { assets, cover_image_url, layout, link: { url } } = unit
```
Comment (kanaabe) [inline src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: Don't we need to pull this in from the panel?

Comment (damassi) [reply src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: Ahh I see: `this.props.display.canvas` is what `unit` resolves to further up the chain

Comment (kanaabe) [reply src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: Nevermind, discussed irl!

Comment (damassi) [reply src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: As we talked about offline, gonna patch this at the force query level

**Pull request merged**